### DISCUSSION TITLE
Cross-User shared/global Prompts

### DISCUSSION THREAD

**Discussion by notdefine** [danny-avila/LibreChat#4752] [Feature Requests & Suggestions]
Hi there,

currently, you could share prompts to other users by exporting and importing them. For a company, it would be nice to configure global Prompts, which each user has in its list.

Is that planned?

**Comment by silenceroom** [comment]
I echo this feature request. 

This can be a killer feature in a company scenario. I bet many would even love to pay for this.

**Comment by silenceroom** [comment]
Further more, Global Agents would be in the same idea to adding to this feature.

**Reply by dvejsada** [reply]
> Further more, Global Agents would be in the same idea to adding to this feature.

You can share your agents to all users as Admin.

**Comment by dvejsada** [comment]
> Hi there,
> 
> currently, you could share prompts to other users by exporting and importing them. For a company, it would be nice to configure global Prompts, which each user has in its list.
> 
> Is that planned?

You can do that by sharing Prompt template as Admin to all users.

**Reply by notdefine** [reply]
This may be a stupid question, but how do I do it? It is Possible to share Agents, but I dont this Option for my Admin user.

**Reply by notdefine** [reply]
Okay, it was hidden because Admins wer not allowed to Share Agent. 

![grafik](https://github.com/user-attachments/assets/ede31296-2842-4307-8ccc-3d18721b8a64)
